testing/mm/resmonitor/Make.defs

```diff
@@ -1,5 +1,5 @@
 ############################################################################
-# apps/testing/resmonitor/Make.defs
+# apps/testing/mm/resmonitor/Make.defs
```

Comment:
this is a tool

Reply:
maybe we should move already tools/utils to `apps/system` not to double the work in future ?